openedx/core/djangoapps/credit/signals.py

```diff
@@ -18,3 +21,27 @@ def listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable=
     from .tasks import update_credit_course_requirements
 
     update_credit_course_requirements.delay(unicode(course_key))
+
+
+@receiver(MIN_GRADE_REQUIREMENT_STATUS)
+def listen_for_grade_calculation(sender, request, grade_summary, course_key, deadline, **kwargs):  # pylint: disable=unused-argument
+    """Receive 'MIN_GRADE_REQUIREMENT_STATUS' signal and update minimum grade
+    requirement status.
```

Comment:
Please update this docstring to include a description of the arguments and keyword arguments.